### PR TITLE
fix link and packages in `authentication.mdx`

### PR DESCRIPTION
**Pull request by jsparkdev**
#### Related issues & labels (optional)

- fix incorrect link
- add missing packages from manual installation

### CODE REVIEW

File: src/content/docs/en/guides/authentication.mdx [hunk 1]
```diff
@@ -91,17 +91,17 @@ To install `astro-auth` manually, install the required package for your package
...
   <Fragment slot="npm">
   ```shell
-  npm install auth-astro
+  npm install auth-astro @auth/core
   ```
   </Fragment>
   <Fragment slot="pnpm">
   ```shell
-  pnpm add auth-astro @auth/core
+  pnpm add auth-astro @auth/core cookie
```
Comment (Mayank-Malviya) [inline src/content/docs/en/guides/authentication.mdx:99]: I am seeing this `cookie` package for the first time, so I have to ask doesn't pnpm installs it with `@auth/core` like how npm or yarn does it. Other than that, I don't find any issues on my end 👍

Comment (TheOtterlord) [reply src/content/docs/en/guides/authentication.mdx:99]: I'll have to check this, but I remember at some point there were issues on `pnpm` relating to this package. @jsparkdev can you recall what the problem you faced was?

**Review by TheOtterlord — COMMENTED** [review]
Looks good! Thanks! I've left a couple of comments. One about the URL of the Lucia tutorial, and another about the cookie package

File: src/content/docs/en/guides/authentication.mdx [hunk 2]
```diff
@@ -50,7 +50,7 @@ Use [Lucia's "Getting started in Astro"](https://lucia-auth.com/getting-started/
 ### Usage
 
 :::tip
-Follow one of Lucia's complete Astro tutorials to add [username and password authentication](https://lucia-auth.com/tutorials/username-and-password/astro) or [GitHub OAuth](https://lucia-auth.com/tutorials/username-and-password/astro) to your Astro project.
+Follow one of Lucia's complete Astro tutorials to add [username and password authentication](https://lucia-auth.com/tutorials/username-and-password/astro) or [GitHub OAuth](https://lucia-auth.com/tutorials/github-oauth/) to your Astro project.
```
[the remark this answers is not in the document]
Comment (jsparkdev) [reply src/content/docs/en/guides/authentication.mdx:53]: thank you

File: src/content/docs/en/guides/authentication.mdx (hunk 1 above)
Comment (jsparkdev) [reply src/content/docs/en/guides/authentication.mdx:99]: I confirmed that it runs without installing the `cookie` package manually.

However, I have confirmed that it does not work properly in the latest version of `@auth/core` and that `@auth/core@^0.18.6` is required.

**Comment by jsparkdev** [comment]
I just confirmed that it runs without having to install the `cookie` package separately.

However, we have confirmed that it does not work properly in the latest version of `@auth/core` and that `@auth/core@^0.18.6` is required as shown in [`auth-astro` repository](https://github.com/nowaythatworked/auth-astro?tab=readme-ov-file#installation).

So I modified the `@auth/core` library version.

**Review by TheOtterlord — APPROVED** [review]
Sounds good. Thanks for checking! I'll see if I can sort out the compatibility issues with the latest version soon.

**Comment by sarah11918** [comment]
!coauthor

**Pull request merged**